types/node/v16/test/cluster.ts

```diff
@@ -1,5 +1,5 @@
 import { connect } from 'net';
-import cluster, { ClusterSettings, Worker } from 'node:cluster'; // requires synthetic default imports, is this worth it?
+import * as cluster from 'node:cluster'; // requires synthetic default imports, is this worth it?
```

Comment:
why are you changing the imports, they should still work, e.g. this works with your pr branch:
```ts
import cluster  from 'node:cluster';
import type { ClusterSettings, Worker } from 'node:cluster';
```

Reply:
I got some linting errors telling me I wasn't allowed to have two imports from the same source. Let me see if I can track it down.

I had initially used exactly the same syntax you propose but had the linter yelling at me.

Reply:
Here is what I was seeing. IMHO, this lint rule is forcing us to use less-than-ideal syntax and is probably not adding the value we want.
```
2> Error: /Users/ggoodman/Projects/ggoodman/DefinitelyTyped/types/node/test/cluster.ts:3:1
2> ERROR: 3:1  no-duplicate-imports  Multiple imports from 'node:cluster' can be combined into one.
2> 
2>     at testTypesVersion (/Users/ggoodman/Projects/ggoodman/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:194:15)
2>     at async runTests (/Users/ggoodman/Projects/ggoodman/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:170:13)
```

Reply:
oh, yes, that's good code hygiene rule. On DT you can escape this using inline comment.
I understand, that's ok for the change.